### PR TITLE
Feature/andrew/prelim open llm wrappers

### PR DESCRIPTION
**Pull request by ajn2004**
changes endpoints in imports
refactors app directory into src/llm_vm/server
adds functionality for more onsite models

### CODE REVIEW

File: src/llm_vm/server/main.py [hunk 1]
```diff
@@ -29,3 +29,6 @@ def generate_hash(input_string):
     sha256_hash = hashlib.sha256()
     sha256_hash.update(str(input_string).encode('utf-8'))
```
Comment (cartazio) [inline src/llm_vm/server/main.py:30]: why hash functions?

Comment (ajn2004) [reply src/llm_vm/server/main.py:30]: vestigial code, I didn't add any of this

File: src/llm_vm/server/routes.py [hunk 2]
```diff
@@ -2,8 +2,7 @@
 import json
 import os
 import openai
-from app import optimizer
-from src.llm_vm.agents.REBEL import agent
+from llm_vm.agents.REBEL import agent
```
[the remark this answers is not in the document]
Comment (ajn2004) [reply src/llm_vm/server/routes.py:5]: done in main.py to doesn't need to be imported

File: src/llm_vm/server/main.py (hunk 1 above)
Comment (cartazio) [reply src/llm_vm/server/main.py:30]: remove it :)

File: src/llm_vm/server/main.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 CORS(app)
 app.config["DEBUG"] = True
 
-from app import routes
+from llm_vm.server import routes
```
Comment (cartazio) [inline src/llm_vm/server/main.py:21]: why not `import llm_vm.server.routes as routes`

**Pull request merged**